pkg/object/mqttproxy/client.go

```diff
@@ -329,6 +333,7 @@ func pipelineWrapper(fn processFn, packetType PacketType) processFnWithErr {
 
 func processPublish(c *Client, packet packets.ControlPacket) {
 	publish := packet.(*packets.PublishPacket)
+	c.broker.processBrokerModePublish(c.info.cid, publish)
```

Comment:
`processBrokerModelPublish` could have a significant impact on the throughput of the device's publishing operation.  Can we call it in the goroutine? 
I have a question, I think that it might be more efficient to call the function after sending an acknowledgment to the client."

Reply:
yeah, u r right... i will update that...